packages/vaadin-themable-mixin/test/themable-mixin.test.js

```diff
@@ -1,29 +1,39 @@
 import { expect } from '@esm-bundle/chai';
-import { fixtureSync } from '@vaadin/testing-helpers';
-import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
-import { css, registerStyles, ThemableMixin } from '../vaadin-themable-mixin.js';
+import { fixtureSync, nextFrame } from '@vaadin/testing-helpers';
+import { PolymerElement } from '@polymer/polymer/polymer-element.js';
+import { css, registerStyles, ThemableMixin, unsafeCSS } from '../vaadin-themable-mixin.js';
 
 let createStyles =
   window.createStylesFunction ||
   ((moduleId, themeFor, styles) => {
     registerStyles(themeFor, styles, { moduleId });
   });
 
-let defineCustomElement = (name, parentName, content = '') => {
-  class CustomElement extends ThemableMixin(parentName ? customElements.get(parentName) : PolymerElement) {
-    static get is() {
-      return name;
-    }
+let defineCustomElement =
+  window.defineCustomElementFunction ||
+  ((name, parentName, content = '', styles) => {
+    class CustomElement extends ThemableMixin(parentName ? customElements.get(parentName) : PolymerElement) {
```

Comment:
Could also consider extracting the parent element expression into a variable here.

Reply:
Done.